boards/nordic/nrf52840_dongle/src/main.rs

```diff
@@ -56,7 +56,8 @@ static mut PROCESSES: [Option<&'static dyn kernel::procs::ProcessType>; NUM_PROC
     [None; NUM_PROCS];
 
 // Static reference to chip for panic dumps
-static mut CHIP: Option<&'static nrf52840::chip::Chip> = None;
+nrf52840::create_default_nrf52840_peripherals!(Nrf52840Peripherals);
```

Comment:
How can this be invoked outside of a function?

Reply:
It only defines a struct and methods on that struct, it doesn't do anything at runtime.

Reply:
Ok, see my other comment regarding the naming. Let's discuss that there.